assets/js/team.tsx

```diff
@@ -0,0 +1,112 @@
+import { h, render, Fragment } from "preact";
+import { load as loadYAML } from "js-yaml";
+import { Alert } from "./components/alert";
+import { loggedInUser, PrivateLoginLink } from "./private";
+
+interface GithubFile {
+  content: string;
+  encoding: "base64" | string;
+}
+
+interface TeamMember {
+  name: string;
+  github: string;
+  subteam?: string;
+  responsibilities?: string;
+  email?: string | string[];
+  home_orgs?: string;
+  fed?: boolean;
+  aws?: string;
+}
+
+interface Alum {
+  name: string;
+  github: string;
+}
+
+interface TeamRoster {
+  team_members: TeamMember[];
+  alumni: TeamMember[];
+}
+
+function GithubLink({ username }: { username: string }) {
+  return <a href={`https://github.com/${username}`}>@{username}</a>;
+}
+
+function TeamMemberRoster({ members }: { members: TeamMember[] }) {
+  return (
+    <ul>
+      {members
+        .sort(({ name: aName }, { name: bName }) => aName.localeCompare(bName))
+        .map(({ name, github, email, subteam, responsibilities, fed, aws }) => (
+          <li>
```

Comment:
Preact isn't as loud about it as React, but `key`s are still a good practice.

https://preactjs.com/tutorial/08-keys/#the-key-to-list-rendering
https://reactjs.org/docs/lists-and-keys.html#keys

Could be relevant if ever we'd allow sorting or re-updating of the list.

```suggestion
          <li key={email}>
```

Reply:
fab1062797b609c8c637781b47b942927ed6e75a